src/parser.cpp

```diff
@@ -342,12 +342,13 @@ void Parser::ParserImpl::loadModel(const ModelPtr &model, const std::string &inp
                 // and issue-checked in loadEncapsulation().
                 encapsulationNodes.push_back(componentRefNode);
             } else {
-                // TODO Should this be removed?
+                // Empty encapsulations are valid, but may not be intended.  This issue has been downgraded to a warning.
```

Comment:
No need for the "This issue has been downgraded to a warning." part of the comment.

Reply:
Done in f5e36a72